tests/testthat/test-step_naomit2.R

```diff
@@ -0,0 +1,30 @@
+library(dplyr)
+library(epiprocess)
+library(parsnip)
+library(workflows)
+
+# Random generated dataset
+set.seed(100)
+x <- tibble(geo_value = rep("nowhere",200),
+            time_value = as.Date("2021-01-01") + 0:199,
+            case_rate = rpois(100,20) + 1:200,
+            death_rate = rpois(100,10) + 1:200) %>%
+  as_epi_df()
+
+# Preparing the datasets to be used for comparison
+r <- epi_recipe(x) %>%
+  step_epi_ahead(death_rate, ahead = 7) %>%
+  step_epi_lag(death_rate, lag = c(0,7,14))
+
+z1 <- step_naomit2(r)
+z2 <- r %>%
+  step_naomit(all_predictors()) %>%
+  step_naomit(all_outcomes(), skip = TRUE)
+
+# Test
+test_that("Check that both functions behave the same way", {
+  expect_identical(z1$steps[[3]][-1][-5],z2$steps[[3]][-1][-5])
```

Comment:
Just out of curiosity, would it make sense to just call `expect_equal(z1, z2)` so that it covers the tests of the class type and the values in the tibble.

Reply:
I tried that, but it doesn't work. The reason being is that each z1 and z2 contain some of their own unique identifiers, namely the quosure and the $id.

Reply:
We should at least add some comments here to explain why you need to grab these elements.

Reply:
I just did.